.github/ISSUE_TEMPLATE/Bug_report.yml

```diff
@@ -52,76 +20,42 @@ body:
-      validations:
-          required: false
-
-    - type: textarea
-      attributes:
-          label: Code snippet (optional)
-          description: |
               If this bug is to related to a developer API, please share a code snippet that demonstrates the issue. For small snippets paste it directly here, or you can use GitHub Gist to share multiple code files: https://gist.github.com
-              Please ensure the shared code can be used by a developer to reproduce the issue—ideally it can be copied into a local development environment or executed in a browser console to help debug the issue.
+              Please ensure the shared code can be used by a developer to reproduce the issue—ideally it can be copied into a local development environment or executed in a browser console to help debug the issue
       validations:
           required: false
 
     - type: textarea
       attributes:
-          label: WordPress Information
-          description: |
-              Please list what WordPress version you are using. You can find this information in Tools → Site Health → Info → WordPress
-      validations:
-          required: false
-
-    - type: textarea
-      attributes:
-          label: Gutenberg Information
+          label: Environment info
           description: |
               Please list what Gutenberg version you are using. If you aren't using Gutenberg, please note that it's not installed.
+          placeholder: |
+            - WordPress version, Gutenberg version.
+            - Browser(s) are you seeing the problem on.
+            - Device you are using and operating system (e.g. "Desktop with Windows 10", "iPhone with iOS 14", etc.)
       validations:
           required: false
-
-    - type: dropdown
-      id: browsers
+          
+    - type: checkboxes
       attributes:
-          label: What browsers are you seeing the problem on?
-          multiple: true
+          label: Pre-checks
+          description: Please check if the bug has already been reported by searching https://github.com/WordPress/gutenberg/issues and make sure before reporting the bug is neither related to another plugin nor theme specific.
           options:
-              - Firefox
-              - Chrome
-              - Safari
-              - Microsoft Edge
-              - Other
-
-    - type: textarea
-      attributes:
-          label: Device Information
-          description: |
-              Please list what device you are using e.g. "Desktop" or "iPhone 11".
-      validations:
-          required: false
-
-    - type: textarea
-      attributes:
-          label: Operating System Information
-          description: |
-              Please list what operating system you are using e.g. "Windows 10" or "iOS 14"
-      validations:
-          required: false
+              - label: I have searched the existing issues.
```

Comment:
Should these remain required?

Reply:
Good call, I was thinking of them as "I accept the terms", but we probably don't need to enforce them, just remind them 👍 (like with the PR checklist)